index.html

```diff
@@ -3,14 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
-    <title>My first Microverse student project</title>
-
+    <title>Luis's portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link rel="stylesheet" href="css/style.css">
   </head>
-  <body>
 
-    <h1>Hello Microverse!</h1>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <a href="#home" class="nav-logo">My Logo</a>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact" class="nav-link">Contact</a>
+          </li>
+        </ul>
+        <div class="mobile-menu"></div>
+      </nav>
+    </header>
 
+    <section class="main">
+      <h1 class="intro">
+        I'm Luis<br>Glad to see you!
+      </h1>
```

Comment:
- [ ] **REQUIRED:** Also the colour of the main title is not the same as the given template, I offer you to pick all colours the same, for this element it is `#172B4D`